### PR TITLE
Hello world size regression

### PR DESCRIPTION
**Pull request by mhevery**
These changes shave about 600 bytes from the minified output.

### CODE REVIEW

File: packages/core/src/render3/instructions.ts [hunk 1]
```diff
@@ -298,10 +300,10 @@ export function renderTemplate<T>(
         null);
 
     const componentTView =
-        getOrCreateTView(templateFn, consts, vars, directives || null, pipes || null, null);
+        createTView(-1, templateFn, consts, vars, directives || null, pipes || null, null);
```
Comment (kara) [inline packages/core/src/render3/instructions.ts]: IIUC, this means every time we call `renderTemplate()` with the same template function, we'll be processing it as if it's the first time?

**Review by kara — COMMENTED** [review]
Looks like we need to run the linter and the symbol updaters, but seems reasonable to me. Just one question about `renderTemplate()`

File: packages/core/src/render3/instructions.ts (hunk 1 above)
Comment (JoostK) [reply packages/core/src/render3/instructions.ts]: @kara Yes, but `renderTemplate` is called form tests only.

I am actually wondering where the `TView` of embedded templates is now attached to, as embedded templates don't have a `ComponentDef` AFAIK.

Comment (kara) [reply packages/core/src/render3/instructions.ts]: @JoostK At the moment, yes. But it was always intended to be a public API, not a testing utility.

`TViews` for embedded templates are stored in the host `TNode` (`TNode.tViews`).

**Review by kara — APPROVED** [review]
LGTM (aside from test failures)

File: packages/core/src/render3/instructions.ts (hunk 1 above)
Comment (kara) [reply packages/core/src/render3/instructions.ts]: Update: Talked to @mhevery, and we're going to table the "public API" version of this for now. We need to re-think the design, so the perf doesn't matter for the time being.

**Pull request closed without merging**